### ISSUE TITLE
NaN appear in Padding, when using history

### ISSUE THREAD

**Issue by romainVala** [fepegar/torchio#376]
**🐛Bug**

Hi  
a small one, but a funny one, related to how the internal change of the padding_mode (from int to string) is not clear enough ... for history !

**To reproduce**

```python
# Your code here
import torchio as tio
s=tio.datasets.Colin27()
t=tio.CropOrPad(target_shape=[182,218,182] ,padding_mode =4)
st=t(s)
st_again=st.history[0](s)
print('what is the padding default? {} not {}'.format(st.t1.data[0,0,0,0],st_again.t1.data[0,0,0,0]))

```

```python-traceback
# Your errors here
what is the padding default? 4.0 not nan
st.history
Out[6]: [Pad(padding=(1, 0, 1, 0, 1, 0), padding_mode=constant)]

```

**Expected behavior**

**Actual behavior**

**TorchIO version**
